arbitrum-docs/launch-orbit-chain/orbit-sdk/how-tos/configuring-orbit-chain-with-sdk.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'How to configure an Orbit chain using the Orbit SDK'
+sidebar_label: 'Orbit Chain Configuration'
+description: 'How to configure an Orbit chain using the Orbit SDK '
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 6
+---
+
+This guide will explore the essentials of configuring your Orbit chain. Configuration includes a range of settings from the parent chain and node configuration to specific child chain parameter configurations. Our focus here is on child chain configuration, guiding you through the necessary adjustments to customize your chain's operation.
+
+In the introductory section, we outlined three primary configurations required for setting up and managing an Orbit chain. These configurations play a crucial role in ensuring the efficient operation of your chain. Let's delve into each type:
+
+## 1. Parent Chain Configuration
+Configuring the parent chain is an essential initial step in setting up your Orbit chain. Most of these configurations are specified during the setup phase. Detailed instructions can be found in the [Rollup Deployment Parameters](deploying-rollup-chain-with-sdk#rollup-deployment-parameter) section of the rollup deployment guide. 
+
+After the initial setup, the chain owner can modify configurations as needed. For instance, the validator set can be updated by invoking the [`setValidKeyset`](https://github.com/OffchainLabs/nitro-contracts/blob/90037b996509312ef1addb3f9352457b8a99d6a6/src/bridge/SequencerInbox.sol#L751) function with a new set of validators. This adaptability facilitates continuous optimization and management of the chain.
+
+## 2. Node/Chain Configuration
+This category includes settings adjustable within the `nodeConfig.json` file, directly impacting the operation of the chain's nodes, including special nodes like validators and sequencers. These settings are vital for tailoring the node's functionality to specific requirements or performance criteria. The chain owner can modify these configurations during the node config generation process, ensuring that each node operates with the desired settings from the start. For more information, refer to the [Node Configuration Preparation](preparing-node-config-with-sdk.md) documentation.
+
+## 3. Child Chain Parameter Configuration
```

Comment:
```suggestion
## 3. Child chain parameter configuration
```

nit - sentence-case titles of all kinds ([ref](https://docs.arbitrum.io/for-devs/contribute#style-conventions))